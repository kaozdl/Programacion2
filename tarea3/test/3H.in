# caso 3H
liberar_estructuras
insertar_binario (1,Champ_URU) .
insertar_binario (2,URU_F) (3,COL_F) .
insertar_binario (4,PER_SF) (5,URU_SF) (6,ARG_SF) (7,COL_SF) .
cantidad_caminos (1,Champ_URU) (2,URU_F) (5,URU_SF) .
cantidad_caminos (6,ARG_SF) .
cantidad_caminos (3,COL_F) (7,COL_SF) .
remover_binario PER_SF .
insertar_binario (4,ECU_SF) (5,URU_SF) .
filtrar_binario 3 > .
filtrar_binario 4 < .
filtrar_binario 1 = .
kesimo_subarbol 1 .
kesimo_subarbol 3 .
kesimo_subarbol 2 .
kesimo_subarbol 4 .
kesimo_subarbol 6 .
kesimo_subarbol 7 .
kesimo_subarbol 5 .
buscar_subarbol Champ_URU .
buscar_subarbol URU_SF .
buscar_subarbol COL_SF .
buscar_subarbol COL_F .
buscar_subarbol ARG_F .
buscar_subarbol Champ_CHI .
Fin
