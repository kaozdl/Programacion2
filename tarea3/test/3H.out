1>
2>3>
(1,Champ_URU)
0
0
4>
-(2,URU_F)
(1,Champ_URU)
-(3,COL_F)
1
1
5>
--(5,URU_SF)
-(2,URU_F)
--(4,PER_SF)
(1,Champ_URU)
--(7,COL_SF)
-(3,COL_F)
--(6,ARG_SF)
2
3
6>1
7>0
8>0
9>
(5,URU_SF)
-(2,URU_F)
(1,Champ_URU)
--(7,COL_SF)
-(3,COL_F)
--(6,ARG_SF)
10>
--(5,URU_SF)
-(2,URU_F)
--(4,ECU_SF)
(1,Champ_URU)
--(7,COL_SF)
-(3,COL_F)
--(6,ARG_SF)
2
3
11>
--(5,URU_SF)
-(4,ECU_SF)
(7,COL_SF)
-(6,ARG_SF)
12>
-(2,URU_F)
(1,Champ_URU)
-(3,COL_F)
13>
(1,Champ_URU)
14>(6,ARG_SF)
15>(7,COL_SF)
16>(3,COL_F)
17>(1,Champ_URU)
18>(2,URU_F)
19>(5,URU_SF)
20>(4,ECU_SF)
21>
--(5,URU_SF)
-(2,URU_F)
--(4,ECU_SF)
(1,Champ_URU)
--(7,COL_SF)
-(3,COL_F)
--(6,ARG_SF)
22>
(5,URU_SF)
23>
(7,COL_SF)
24>
-(7,COL_SF)
(3,COL_F)
-(6,ARG_SF)
25>
26>
27>Fin.
